packages/fetch-engine/src/downloadZip.ts

```diff
@@ -28,7 +28,11 @@ async function fetchChecksum(url: string): Promise<string | null> {
       agent: getProxyAgent(url) as any,
     })
     if (!response.ok) {
-      throw new Error(`Failed to fetch sha256 checksum at ${checksumUrl}. ${response.status} ${response.statusText}`)
+      let errorMessage = `Failed to fetch sha256 checksum at ${checksumUrl}. ${response.status} ${response.statusText}`
+      if (!process.env.PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING) {
+        errorMessage += `\n\nIf you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.\nExample: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1`
+      }
+      throw new Error(errorMessage)
```

Comment:
Note: I thought it would be a good idea to suggest this in the error message to make it discoverable when using Prisma offline.

Reply:
Note: We might change the logic in the future and make this obsolete but in the meantime this would help.